wlog/wrappedlog/wrapped1log/logger_default.go

```diff
@@ -15,18 +15,37 @@
 package wrapped1log
 
 import (
+	"io"
+
 	"github.com/palantir/witchcraft-go-logging/wlog"
+	"github.com/palantir/witchcraft-go-logging/wlog/extractor"
+	"github.com/palantir/witchcraft-go-logging/wlog/reqlog/req2log"
 	"github.com/palantir/witchcraft-go-logging/wlog/svclog/svc1log"
 	"github.com/palantir/witchcraft-go-logging/wlog/trclog/trc1log"
 )
 
 type defaultLogger struct {
 	name        string
 	version     string
+	creator     wlog.LoggerCreator
+	writer      io.Writer
```

Comment:
can you add comments explaining the parameters that are not used by all loggers? when should they be set and what to they control?